rasa/core/cli/run.py

```diff
@@ -20,34 +34,35 @@ def add_run_arguments(parser):
         "--cors",
         nargs="*",
         type=str,
-        help="enable CORS for the passed origin. Use * to whitelist all origins",
+        help="Enable CORS for the passed origin. Use * to whitelist all origins.",
     )
     server_arguments.add_argument(
         "--enable-api",
         action="store_true",
-        help="Start the web server api in addition to the input channel",
+        help="Start the web server api in addition to the input channel.",
     )
-
-    parser.add_argument(
-        "-o",
-        "--log-file",
-        type=str,
-        default="rasa_core.log",
-        help="store log file in specified file",
+    server_arguments.add_argument(
+        "--wait-time-between-pulls",
```

Comment:
I don't think this should be a commandline argument but rather be part of the model server configuration part in `endpoints.yml`